scripts/actions/install_cuda_11.0.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+UBUNTU_VERSION=$(lsb_release -sr)
+UBUNTU_VERSION="${UBUNTU_VERSION//.}"
+
+wget -q https://developer.download.nvidia.com/compute/cuda/repos/ubuntu$UBUNTU_VERSION/x86_64/cuda-ubuntu$UBUNTU_VERSION.pin -O cuda.pin
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/physarumAdv/minds_crawl/pullRequest?prid=6170420)